pkg/koordlet/resmanager/cpu_burst.go

```diff
@@ -513,43 +509,47 @@ func (b *CPUBurst) applyCPUBurst(burstCfg *slov1alpha1.CPUBurstConfig, podMeta *
 			continue
 		}
 
-		if system.ValidateResourceValue(&containerCFSBurstVal, containerDir, system.CPUBurst) {
-			podCFSBurstVal += containerCFSBurstVal
-			ownerRef := executor.ContainerOwnerRef(pod.Namespace, pod.Name, container.Name)
-			containerCFSBurstValStr := strconv.FormatInt(containerCFSBurstVal, 10)
-			updater := executor.NewCommonCgroupResourceUpdater(ownerRef, containerDir, system.CPUBurst, containerCFSBurstValStr)
-			updated, err := b.executor.UpdateByCache(updater)
-			if err == nil {
-				klog.V(5).Infof("apply container %v/%v/%v cpu burst value success, dir %v, value %v",
-					pod.Namespace, pod.Name, containerStat.Name, containerDir, containerCFSBurstVal)
-			} else if system.HostSystemInfo.IsAnolisOS {
-				// cgroup `cpu.burst_us` is expected available on anolis os, and it may not exist in other kernels.
-				klog.Infof("update container %v/%v/%v cpu burst failed, dir %v, updated %v, error %v",
-					pod.Namespace, pod.Name, containerStat.Name, containerDir, updated, err)
-			} else {
-				klog.V(4).Infof("update container %v/%v/%v cpu burst ignored on non Anolis OS, dir %v, "+
-					"updated %v, info %v", pod.Namespace, pod.Name, containerStat.Name, containerDir, updated, err)
-			}
+		podCFSBurstVal += containerCFSBurstVal
+		containerCFSBurstValStr := strconv.FormatInt(containerCFSBurstVal, 10)
+		updater, err := resourceexecutor.GetCgroupUpdater(system.CPUBurstName, containerDir, containerCFSBurstValStr)
+		if err != nil {
+			klog.V(4).Infof("get cpu burst updater for container %s/%s/%s failed, err: %v",
```

Comment:
will not-exist/not-support error can be ignored now?

Reply:
done. the log level is incremented as `.V(5)`